drivers/SmartThings/zwave-switch/src/qubino-switches/init.lua

```diff
@@ -54,7 +54,7 @@ local function getDeviceProfile(device, isTemperatureSensorOnboard)
 end
 
 local function can_handle_qubino_flush_relay(opts, driver, device, cmd, ...)
-  return device.zwave_manufacturer_id == constants.QUBINO_MFR
+  return device:id_match(constants.QUBINO_MFR)
```

Comment:
did this method get fixed to handle nil arguments? I think guess it did, right?

Reply:
It did.